framework/include/utils/MathUtils.h

```diff
@@ -19,6 +19,8 @@
 namespace MathUtils
 {
 
+constexpr Real sqrt2 = 1.4142135623730951;
```

Comment:
`std::sqrt` isn't `constexpr`!? madness

Reply:
Yep

Reply:
I think this needs a bit more discussion. According to my investigations using godbolt.org all versions of GCC and clang evaluate `std::sqrt(2.0)` at compile time and just insert a constant value. 

_However_, you still cannot use `std::sqrt()` inside a `constexpr` function. This will - in accordance with the standard - result in a compile error. We could use this cool compile time newton raphson template meta program: https://gist.github.com/alexshtf/eb5128b3e3e143187794 for a true constexpr sqrt though...